components/gc-subway/section-end-fr.html

```diff
@@ -1,58 +1,42 @@
 ---
 {
-	"title": "[Page 1] - GC menu par carte de métro (provisoire)",
+	"title": "[Page 1] - GC Métro - Exemple de page de section avec élément d'arrêt de section",
 	"language": "fr",
-	"altLangPage": "gc-subway-en.html",
-	"pageclass": "cnt-wdth-lmtd",
+	"altLangPage": "section-end-en.html",
 	"layout": "without-h1",
-	"dateModified": "2022-08-25",
+	"dateModified": "2023-01-04",
 	"share": "true"
 }
 ---
-{% include alert-provisional.html provisionaldir="../" %}
 
-<ul class="list-inline">
-	<li><a class="btn btn-primary" href="docs-fr.html">Documentation</a></li>
-	<li><a class="btn btn-primary" href="https://github.com/wet-boew/GCWeb/issues/new?title=gc-subway">Questions ou commentaires?</a></li>
-</ul>
+{%- include gc-subway/docs-questions.html -%}
 
-<nav class="provisional gc-subway">
-	<h1 id="gc-document-nav">[Nom du service]</h1>
+<nav class="gc-subway provisional" data-gc-subway='{"subtitle": "My sections"}'>
```

Comment:
```suggestion
<nav class="gc-subway provisional">
```

Is `data-gc-subway` value can be extracted or it need to be provided by the author?

Reply:
Est-ce qu'on peut modifié le design pattern pour l'inclure dans le HTML à la place d'une configuration

Example/idée lorsque Enhanced:
```
<h1 class="h3"> <span class="hidden-md visible-xs">[Service name]</span> <span class="landmark-name hidden-xs visible-md visible-lg mrgn-tp-0">Sections</span></h1>
```


Example/idée lorsque coded by author:
```
<h1>[Service name] <span class="landmark-name">Sections</span></h1>
```
où que la class "landmark-name" serait caché par défaut (invisible)

Je sais qu'on a une demande pour identifier les landmark, peut-être: (Id or attribute+JS)
```
<nav aria-labeledby="gc-subway-landmark-name">
    <h1>[Service name] <span id="gc-subway-landmark-name" data-landmark>Sections</span></h1>
</nav>
```

Reply:
Should that title be connected to the h1 instead of the nav? That is essentially an alternate title of when it get combined with the other h1.